src/index.html

```diff
@@ -4,10 +4,229 @@
+                  Home
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about-us">
+                  About us
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#gallery">
+                  Gallery
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#consultation">
+                  Consultation
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#testimonials">
+                  Testimonials
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts">
+                  Contacts
+                </a>
+              </li>
+            </ul>
+            <p class="nav__phone">
+              +1 234 555-55-55
+            </p>
+            <a class="nav__link nav__link--phone" href="tel: +12345555555">
+              Call to order
+            </a>
+          </div>
+        </nav>
+        <img
+          class="header__logo logo"
+          src="./images/Union.png"
+          alt="Logo"
+        >
+        <a
+          class="header__linkTel"
+          href="tel:+12345555555"
+        >
+        </a>
+      </div>
+      <h1 class="header__title">Miami<br>Condo Kings</h1>
+    </header>
+    <main class="page__content">
+      <section
+        class="aboutUs"
+        id="about-us">
+        <h2 class="visually-hidden">About us</h2>
+        <h3 class="aboutUs__title">
+          Miami best<br>Real estate
+          <a class="aboutUs__link" href="">here</a>
+        </h3>
+        <div class="aboutUs__content">
+          <p class="aboutUs__paragraph">
+            Over 50 years experience in love.
+          </p>
+          <p class="aboutUs__paragraph">
+            Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+          </p>
+          <p class="aboutUs__paragraph">
+            The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information about the profitable deals.
+          </p>
+          <p class="aboutUs__paragraph">
+            We will never be late!
+          </p>
+          <p class="aboutUs__paragraph">
+            Guaranteed secure transaction - 100%
+          </p>
+        </div>
+      </section>
+      <section
+        class="gallery"
+        id="gallery">
+        <h2 class="sectionTitle">
+          Gallery
+        </h2>
+        <div class="gallery__item">
```

Comment:
```suggestion
        <div class="gallery__item--wide-left">
```
You can use modifiers `item--wide-left` and `item--wide-right` for your blocks and not overweight your code with :nth-childs and :nth-of-type.
Just use smth like 
`grid-template-columns: 1fr 1fr;` for central block
and
`grid-template-columns: 1fr 2fr;` and `grid-template-columns: 2fr 1fr;` for other blocks